src/main/config/codelist/local/thesauri/theme/GC_Resource_Formats.rdf

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+         xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
```

Comment:
This seems like a type.  I don't see **rdfs** used below but I do see **rdf**.  I think you need to drop the **s**.

Reply:
Sorry just saw the rdf namespace above.   I still don't see where rdfs is being used.  I guess we can keep it.

Reply:
Yeah. The RDFs is from FGP as copy pasted. It will probably ok to remain. I will go replace the SKOs by NS2